ynput/OpenPype
# Color Management- added color management support for simple expected files on Deadline

## Changelog Description
Running of `ExtractOIIOTranscode` during Deadline publish was previously implemented only on DCCs with AOVs (Maya, Max).
This PR extends this for other DCCs with flat structure of expected files.


## Testing notes:
1. Configure Color Management for your DCC of your choice
2. Publish to DL, check if representation were properly color modified
